frontend/public/components/dashboard/top-consumers-card/utils.tsx

```diff
@@ -0,0 +1,87 @@
+import { get } from 'lodash';
+
+const BYTE_UNITS_CONFIG: ByteUnitsConfig = {
```

Comment:
Can we use the existing utilities we have in units.js? If they aren't sufficient, we should add to those.